Match.java

```diff
@@ -46,11 +56,14 @@ public void setGoalConceded(int goalConceded){
 	}
 	
 	public void infoAboutOneGame(){
-		System.out.println(season + ", " + nameOfGame + " " + goalScored + ":" + goalConceded);
+		if (isGameAtHome == true)
+			System.out.println(season + ", " + nameOfGame + " " + goalScored + ":" + goalConceded);
+		else
+			System.out.println(season + ", " + nameOfGame + " " + goalConceded + ":" + goalScored);
 	}
-	
+
 	public static void main (String [] args){
-		Match one = new Match("2019/20", "Shakhtar Donetsk-Manchester City", 0, 3);
+		Match one = new Match("2019/20", "Shakhtar Donetsk-Manchester City", true, 0, 3);
```

Comment:
Think about nameOfGame, I don't think we need it with such meaning, we already have variable responsible for game place(home/away).